tests/test_tui_publish_workflow_security.py

```
@@ -1437,5 +1616,687 @@ def test_stable_release_builds_and_tests_once_before_dispatching_publishers() ->
+        environment.update(
+            {
+                "GITHUB_REPOSITORY": "manaflow-ai/cmux",
+                "PATH": f"{bin_dir}:{environment['PATH']}",
+                "POLL_INTERVAL_SECONDS": "0",
+                "WAIT_TIMEOUT_SECONDS": "30",
+                "CLOCK_SKEW_SECONDS": "5",
+                "STATE": str(state),
+                "DETAIL_STATE": str(detail_state),
+            }
+        )
+        result = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)